core/src/main/java/io/grpc/util/RoundRobinLoadBalancerFactory.java

```diff
@@ -99,6 +104,8 @@ public LoadBalancer newLoadBalancer(LoadBalancer.Helper helper) {
     @VisibleForTesting
     static final Attributes.Key<Ref<ConnectivityStateInfo>> STATE_INFO =
         Attributes.Key.create("state-info");
+    static final Attributes.Key<Ref<PickResult>> PICK_RESULT = Attributes.Key.create("pick-result");
+    static final Attributes.Key<Ref<Subchannel>> STICKY_REF = Attributes.Key.create("sticky-ref");
```

Comment:
Actually it's not used directly by tests. I left it package-private to avoid synthetic access from `StickinessState` inner class. I could make it `private` though if you like, or just add a comment to make that explicit?

Reply:
Adding a comment "package-private to avoid synthetic access" would be nice.